<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDetailsWidget</class>
 <widget class="QWidget" name="ImageDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: rgb(255, 255, 255);
	border-radius: 15px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>22</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="detailsDetail" columnstretch="1,8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QLabel" name="filepathValue">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>D:CUCEI3RO_INFOPOOFiguresDetectionrestest.jpg</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="filepathLabel">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Ruta del archivo:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Nombre del archivo: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="sizeLabel">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Info de tama√±o: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="sizeValue">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(0, 0, 0);
}
</string>
          </property>
          <property name="text">
           <string>300 x 300     60 KB     96 dpi     8 bit</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="nameValue">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>poligonos</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
